frontend/cypress/integration/featureFiles/istio_config.feature

```diff
@@ -0,0 +1,76 @@
+Feature: Kiali Istio Config page
+
+  User opens the Istio Config page and sees the Istio Config objects for Bookinfo
+
+  Background:
+    Given user is at administrator perspective
+    And user opens the "istio" page
+    And user selects the "bookinfo" namespace in the NamespaceSelector
+    And Kiali is done loading
+
+  @istio-page
+  Scenario: See a table with correct info
+    Then user sees a table with headings
+      | Name | Namespace | Type | Configuration | 
+    And the "bookinfo-gateway" row is visible
+    And the "Name" column on the "bookinfo-gateway" row has a link ending in "/namespaces/bookinfo/istio/gateways/bookinfo-gateway"
+    And the "Namespace" column on the "bookinfo-gateway" row has the text "bookinfo"
+    And the "Type" column on the "bookinfo-gateway" row has the text "Gateway"
+    And the "Configuration" column on the "bookinfo-gateway" row has a link ending in "/namespaces/bookinfo/istio/gateways/bookinfo-gateway?list=yaml"
+
+  @istio-page
+  Scenario: Filter istio config table by Istio Name
+    When user selects filter "Istio Name"
+    And user filters for name "bookinfo-gateway"
+    Then user sees "bookinfo-gateway" in the table
+    And table length should be 1
+  
+  @istio-page
+  Scenario: Filter istio config table by Istio Type
+    When user selects filter "Istio Type"
+    And user filters for istio type "Gateway"
+    Then user sees "Gateway" in the table
+    And table length should be 1
+  
+  @istio-page
+  Scenario: Filter istio config table by Config
+    When user selects filter "Config"
+    And user filters for config "Valid"
+    Then user sees "bookinfo-gateway" in the table
+    And table length should be 2
+
+  @istio-page
+  Scenario: Navigate to the create AuthorizationPolicy page through the actions dropdown
+    When the user clicks the actions button
+    And the user clicks the create "AuthorizationPolicy" action
+    Then the user navigates to the "/istio/new/AuthorizationPolicy?namespaces=bookinfo" page
```

Comment:
It is a little late here, so as a preliminary review, I think all `Navigate*` scenarios are somewhat procedural. 
Suggestions in the ["Writing better Gherkin" docs](https://cucumber.io/docs/bdd/better-gherkin/) gives the understanding that we shouldn't be referring to the specific controls in the application (i.e. the button nor the actions/items in the dropdown).

I may have a few slips in my Logs PR (ouch).

In general, I understand the idea behind Gherkin (and BDD), is that the scenarios should describe behavior. The `Navigate to the create AuthorizationPolicy page through the actions dropdown` sentence is more on the  implementation details side IMHO.

I'm sorry if I'm being picky. All this is very new for me, so I'm heavily following suggestion of all docs I've read. But if we are going to be OK with this style, I can be more loose on the style/language to use.

Reply:
Thanks for the link to the docs. I have not read those and I definitely see now that pretty much all the tests are procedural :). I'll play with the step definitions a bit to try and make it less coupled to the UI implementation. I would like for the step definitions to be able to be re-used easily though. Writing the tests for the apps page was really easy and required writing zero code because I could re-use existing step definitions in the feature file.

Reply:
> I would like for the step definitions to be able to be re-used easily though.

Yes, that's why I said that if we are OK with this style, I won't see wrong to sometimes break the rules in favor of reusability. However, I also like that idea about writing Gherkin that won't change if implementation changes. We should be smart choosing where/when to do tradeoffs.

Reply:
Updated with some more "gherkin friendly" syntax. Hopefully reads better now. Thanks for the feedback!